src/view-backend-exportable-fdo-egl.cpp

```diff
@@ -118,6 +118,17 @@ class ClientBundleEGLDeprecated final : public ClientBundle {
         assert(!"should not be reached");
     }
 
+    struct wpe_dmabuf_pool_entry* createDmabufPoolEntry() override
+    {
+        assert(!"should not be reached");
```

Comment:
I would go for `g_assert_not_reached()` here and in the instances below, WDYT?

```suggestion
        g_assert_not_reached();
```

Reply:
This is aligned with what the other dead-end overrides do.